### PR TITLE
Added security notes for the multikey setup

### PR DESCRIPTION
**Pull request by iulianpascalau**
#### Description of the pull request (what is new / what has changed)
- added security notes for the multikey setup

#### Did you test the changes locally ?
- [X] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [X] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/validators/key-management/multikey-nodes.md [hunk 1]
```diff
@@ -152,3 +152,21 @@ example-0 e296e97524483e6b59...
...
 random-0  791c7e2bd6a5fb1371...
 ```
+
+### Security notes for the multikey nodes
+
+As stated above, the multikey feature is able to use any number of keys with just 4 nodes. 
+At the first sight, this can be seen as a security degradation in terms of means of attacking a large staking provider but there are ways to mitigate these concerns as explained in the following list:
+1. use the recommendation found in this page regarding the maximum number of keys per multikey group;
+2. for each main multikey group use at least one backup multikey group in case something bad happens with the main group;
+3. use the NamedIdentity configuration explained above to hide the BLS keys and their declared identity from the actual nodes that manages the keys.
```
Comment (sstanculeanu) [inline docs/validators/key-management/multikey-nodes.md:162]:
```suggestion
3. use the NamedIdentity configuration explained above to hide the BLS keys and their declared identity from the actual nodes that manage the keys.
```

File: docs/validators/key-management/multikey-nodes.md [hunk 2]
```diff
@@ -152,3 +152,21 @@ example-0 e296e97524483e6b59...
...
+
+Regarding point 3, each managed BLS key will create a virtual p2p identity that no node from the network can connect to since it does not advertise the connection info but is only used to sign p2p messages.
+Associated with a separate named identity, the system will make that BLS key virtually unreachable, and it's origin hidden from the multikey nodes. For this to work properly, the node operators will need to apply the following changes on the `prefs.toml` file:
```
Comment (sstanculeanu) [inline docs/validators/key-management/multikey-nodes.md:165]:
```suggestion
Associated with a separate named identity, the system will make that BLS key virtually unreachable, and its origin hidden from the multikey nodes. For this to work properly, the node operators will need to apply the following changes on the `prefs.toml` file:
```

Comment (bogdan-rosianu) [inline docs/validators/key-management/multikey-nodes.md:165]: it's -> its
For this to work properly -> Therefore

File: docs/validators/key-management/multikey-nodes.md (hunk 1 above)
Comment (bogdan-rosianu) [inline docs/validators/key-management/multikey-nodes.md:162]: NamedIdentity -> \`NamedIdentity\`

and maybe find a replacement for 'hide'.

Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:162]: done

File: docs/validators/key-management/multikey-nodes.md (hunk 2 above)
Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:165]: done

Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:165]: done

File: docs/validators/key-management/multikey-nodes.md (hunk 1 above)
Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:162]: done, replaced `hide` with `obfuscate`

**Review by bogdan-rosianu — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**